docs/en/observability/logs-checklist.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[logs-checklist]]
+= Logs resource guide
+
+In this guide, you'll find resources on sending log data to {es}, configuring your logs, and analyzing your logs. 
+
+[discrete]
+[[logs-getting-started-checklist]]
+== Get started with logs
+
+If you're new to ingesting, viewing, and analyzing logs with Elastic, see <<logs-metrics-get-started, Get started with logs and metrics>> for an overview of adding integrations, installing and running an {agent}, and monitoring logs.
+
+[discrete]
+[[logs-send-data-checklist]]
+== Send data to {es}
+
+You can send data to {es} in different ways depending on your needs: 
+
+- {agent}
```

Comment:
Eventually we should get rid of this and just tell users what to use per use case.